wger/weight/templates/weight_overview.html

```diff
@@ -38,6 +38,24 @@
 <div id="weight_diagram"></div>
 
 {% if is_owner %}
+{% if last_five_weight_entries_details %} 
+<table border="1" style="width:100%">
+    <tr>
+        <th>Date</th>
```

Comment:
Better write ``{% trans "Date" %}``, then the header can be translated when using the site on another language, same with change, etc.